openrl/envs/common/build_envs.py

```diff
@@ -36,13 +36,22 @@ def _make_env() -> Env:
                 new_kwargs["env_num"] = env_num
             if id.startswith("ALE/") or id in gym.envs.registry.keys():
                 new_kwargs.pop("cfg", None)
```

Comment:
Envpool can not be installed on many platforms (such as Windows), so you should use the envpool as an external env which is out of OpenRL. Moreover, with the envpool as an external env, you don't need to write test code for codecov, because we only track the code in the openrl folder.

Examples:
- https://openrl-docs.readthedocs.io/en/latest/custom_env/index.html#moreexamples1
- https://github.com/OpenRL-Lab/openrl/tree/main/examples/isaac
- https://github.com/OpenRL-Lab/openrl/tree/main/examples/smac